### DISCUSSION TITLE
POI name

### DISCUSSION THREAD

**Discussion by donnje** [organicmaps/organicmaps#5563] [Q&A] [answered]
Hi,
Is there any way to read the POI name when opening the map?
thanks

**Comment by biodranik** [comment]
Can you please clarify your question? What are you talking about?

**Comment by donnje** [comment]
the POI icon is displayed on the map but not its name, it is only seen in the list

**Reply by biodranik** [reply]
Are you speaking about bookmarked objects/bookmarks? Or about any map objects (POIs)? Most map POIs already display their names.

**Comment by donnje** [comment]
I speaking about bookmarks (personal POI)

**Reply by biodranik** [reply, marked as answer]
Please follow this issue: https://github.com/organicmaps/organicmaps/issues/3164
